src/app/services/http.service.ts

```diff
@@ -18,21 +15,23 @@ export class HttpService {
   //   })
   // };
 
-  constructor(private http: HttpClient) { }
+  constructor(private http: HttpClient) {}
 
   // // use this when making get requests with params
   getRequestWithParams = (endpoint, params = {}) => {
     return this.http.get(this.base_url + endpoint, { params });
+  };
```

Comment:
The semicolon here is not necessary since semicolons are not required after block statements or function declarations. In this case, we have a function declared
```suggestion
  }
```